models/staging/whatsapp-webhook/scr_webhook.yml

```diff
@@ -1,7 +1,7 @@
 version: 2
 
 sources:
-  - name: webhook
```

Comment:
Can we keep it webhook only? or whatsapp_webhook? making it env specific may confuse in future